test/instrumentation/modules/http/outgoing.js

```diff
@@ -135,8 +135,12 @@ function echoTest (type, handler) {
         res.resume()
       })
 
-      var traceparent = req.getHeader('elastic-apm-traceparent')
-      t.ok(traceparent, 'should have elastic-apm-traceparent header')
+      var traceparent = req.getHeader('traceparent')
+      t.ok(traceparent, 'should have traceparent header')
+      if (agent._conf.useElasticTraceparentHeader) {
```

Comment:
This will as far as I can see never be set in this test right? How about creating a test where you specifically test that this config option has the desired effect?

Reply:
The default for `useElasticTraceparentHeader` config is `true` so it will be set, unless I'm missing something. I'm not against having a separate test though, just want to make sure we're on the same page.

Reply:
I was more thinking of testing that if you set it to `false`, then it works as expected. In its current state, the if-statement will _always_ execute as the config-option is _always_ true. But we don't have a test to test that the header is not present if the config option is set to `false` as far as I can see

Reply:
I've added the tests, please have another look.